packages/node_modules/@node-red/runtime/lib/telemetry/index.js

```diff
@@ -48,11 +51,14 @@ async function report () {
     }
     // If enabled, gather metrics
     const metrics = await gather()
-    console.log(JSON.stringify(metrics, null, 2))
+    runtime.log.debug(`Telemetry metrics: ${JSON.stringify(metrics, null, 2)}`)
```

Comment:
```suggestion
```
Rather than 'update to use the proper logging', the intended fix was 'dont log this' 😉

Reply:
It might be interesting to see what is being sent 😁